crypto/bn/bn_prime.c

```diff
@@ -308,9 +308,10 @@ static int bn_is_prime_int(const BIGNUM *w, int checks, BN_CTX *ctx,
         goto err;
 #endif
 
-    ret = ossl_bn_miller_rabin_is_prime(w, checks, ctx, cb, 0, &status);
-    if (!ret)
+    if (ossl_bn_miller_rabin_is_prime(w, checks, ctx, cb, 0, &status) != 1) {
```

Comment:
As the ossl_bn_miller_rabin_is_prime return is boolean could you please keep the original condition?

Reply:
Done!